lib/internal/bootstrap_node.js

```diff
@@ -54,7 +54,7 @@
     NativeModule.require('internal/process/warning').setup();
     NativeModule.require('internal/process/next_tick').setup();
     NativeModule.require('internal/process/stdio').setup();
-    if (browserGlobals) {
+    if (browserGlobals && process.config.v8_enable_inspector) {
```

Comment:
I think the name is actually `process.config.variables.v8_enable_inspector`?

Reply:
You are right, can't believe I missed that. 😞  Will update now. Thanks again!